src/components/Tasks/TasksTable.js

```diff
@@ -56,7 +50,7 @@ const TaskTable = ({ tasks }) => {
     () =>
       filteredTasks.sort(({ dueDate: dateLeft }, { dueDate: dateRight }) =>
         // Sort by date ascending
-        compareAsc(dateLeft, dateRight)
+        dayjs(dateLeft).isAfter(dayjs(dateRight)) ? 1 : -1
```

Comment:
Should we also return `0` if the two dates are identical? Or is it too highly improbable?